### PR TITLE
`Development`: Ignore blank lines in Aeolus scripts

### PR DESCRIPTION
**Pull request by reschandreas**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation, Description, and Context

Currently, the aeolus check would fail since empty lines were manually removed, this PR solves that by ignoring the blank lines. It further removes obsolete docker compose configurations which were not removed when removing all of Bamboo.

### Steps for Testing

1. Run `bash supporting_scripts/generate-aeolus-scripts.sh` and observe that everything is fine, no changes are expected

#### Performance Review
- [x] I (as a reviewer) confirm that the client changes (in particular related to REST calls and UI responsiveness) are implemented with a very good performance 
- [x] I (as a reviewer) confirm that the server changes (in particular related to database calls) are implemented with a very good performance
#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

## Summary by CodeRabbit

- **Chores**
	- Removed configurations related to the `aeolus-bamboo-api` service and adjusted network settings in Docker configurations.
  
- **Refactor**
	- Enhanced script for generating reports by ignoring blank lines during file comparisons.

### CODE REVIEW

**Review by laurenzfb — APPROVED** [review]
Code

**Comment by b-fein** [comment]
Related: #8390

**Review by Strohgelaender — APPROVED** [review]
lgtm. I think we should merge both this and #8390 to have consistency for the current scripts, and ignore whitespace changes in case they happen again in the future.

**Review by mateusmm01 — APPROVED** [review]
code lgtm

File: docker/aeolus.yml [hunk 1]
```diff
@@ -5,22 +5,9 @@ services:
...
-            - aeolus
-            - artemis
-    aeolus-bamboo-api:
-        container_name: aeolus-bamboo-api
-        hostname: aeolus-bamboo-api
-        image: ghcr.io/ls1intum/aeolus/bamboo-generator:nightly
-        command: "--api"
-        pull_policy: always
-        restart: unless-stopped
-        networks:
```
Comment (rstief) [inline docker/aeolus.yml:22]: Is it on purpose that aeolus got removed as network?

**Review by rstief — APPROVED** [review]
LGTM just one question

File: docker/aeolus.yml (hunk 1 above)
Comment (reschandreas) [reply docker/aeolus.yml:22]: yes, with just the one container it's not required anymore.

**Pull request merged**